_posts/2018-01-11-keystone-v3.md

```diff
@@ -25,27 +25,27 @@ overrides.put(KeystoneProperties.KEYSTONE_VERSION, "3");
 
 ### Configuring authentication
 
-Keystone V3 supports several authentication mechanisms that provide authentication tokens with different permissions. It is important to configure the right authentication method, otherwise some operations offered by the Keystone API might not be available.
+Keystone V3 supports several authentication mechanisms, which provide authentication tokens with different permissions. It is important to configure the correct authentication method, otherwise some operations offered by the Keystone API might not be available.
 
-The credentials in Keystone 3 must include the `domain` name and the `username`, as shown in the example above.
-
-By default, jclouds uses **password authentication with unscoped authorization**, although this can be changed by configuring the `KeystoneProperties.SCOPE` property when creating the context, to configure a project or domain authorization scope. For example
+By default, jclouds uses **password authentication with unscoped authorization**. Project or domain authorization scopes can be configured by setting the `KeystoneProperties.SCOPE` property when creating your jclouds context, for example:
 
 {% highlight java %}
 Properties overrides = new Properties();
-// Project scoped authorization (can use the proejct name or the ID)
+// Project scoped authorization (can use the project name or the ID)
 overrides.put(KeystoneProperties.SCOPE, "project:jclouds");
 overrides.put(KeystoneProperties.SCOPE, "projectId:2f9b30f706bc45d7923e055567be2e98");
 // Domain scoped authorization (can use the domain name or the ID)
 overrides.put(KeystoneProperties.SCOPE, "domain:default");
 overrides.put(KeystoneProperties.SCOPE, "domainId:2f9b30f706bc45d7923e055567be2e98");
 {% endhighlight %}
 
-# Using the Keystone V3 APIs
+Credentials in Keystone V3 must include the `domain` name and the `username`, as shown above.
```

Comment:
Not sure what this exactly is referring to - where is `domain` and `username` in the examples?